### ISSUE TITLE
keepAlive

### ISSUE THREAD

**Issue by aicanfly** [TooTallNate/node-socks-proxy-agent#68]
is there any way to enable 'keepAlive: true' to keep connection alive with socks5 proxy server? so I can reuse active socket to get better response times instead of closing connection after every request

it works fine with: https://github.com/mattcg/socks5-https-client but it's not maintained anymore (library doesn't work with newest nodejs version)

**Comment by msklvsk** [comment]
Or, more generally, can we configure the underlying agent with `AgentOptions`?

**Comment by aicanfly** [comment]
any news about this feature?

**Comment by Kikobeats** [comment]
any PR will welcome

**Comment by kevinji** [comment]
This is likely blocked on TooTallNate/node-agent-base#5.

**Comment by TooTallNate** [comment]
This code in this repository has been moved to the [`proxy-agents` monorepo](https://github.com/TooTallNate/proxy-agents), so I am closing this pull request. If you feel that this issue still exists as of the latest release, feel free to open a new issue over there.
